Dockerfile

```diff
@@ -133,23 +137,10 @@ RUN julia -e 'Pkg.add("Lint")'
 RUN luarocks install luacheck
 
 # NPM setup
-# FIXME: we should use package.json from coala
-RUN npm install -g \
-  alex \
-  autoprefixer \
-  bootlint \
-  coffeelint \
-  complexity-report \
-  csslint \
-  dockerfile_lint \
-  eslint \
-  jshint \
-  postcss-cli \
-  remark-cli \
-  tslint \
-  typescript \
-  ramllint \
-  write-good
+# Extract dependencies from coala-bear package.json
+# typescript is a peer dependency
+RUN npm install -g typescript \
+    $(sed -ne '/~/{s/^[^"]*"//;s/".*"~/@/;s/",*//;p}' coala-bears/package.json)
```

Comment:
No.  That installs `coala-bears` globally, and none of the dependencies are global.

There are a few alternatives, all ugly.

> How reliable is this?

We now have bear testing again, so we can see if it breaks.